docs/spec_draft.md

```diff
@@ -846,6 +848,149 @@ defined and one of the following:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.gather
+
+### Semantics
+
+Gathers slices from `operand` tensor from offsets specified in `start_indices`
+and produces a `result` tensor.
+
+The following diagram shows how elements in `result` map on elements in
+`operand` using a concrete example. The diagram picks a few example `result`
+indexes and explains in detail which `operand` indexes they correspond to.
```

Comment:
I know that I proposed this wording, but there's one thing that I overlooked there. It says "indexes", and in the spec we predominantly use "indices". Let's reword?

Reply:
I overlooked it as well. fixed.